### ISSUE TITLE
Request: event for when you've visually started to render a marker during creation

### ISSUE THREAD

**Issue by tarngerine** [ailon/markerjs2#126]
Currently the timeline of creation has some events, but not all:

1. activate the marker / `createNewMarker` -> `markercreating`
2. click/drag on canvas to actually start visually showing the element
3. (for text) typing the text the first time
4. mouseup or click outside (for text) -> `markercreate`, then `markerselect`

in particular I'd love an event for #2, when the user actually mouses down/clicks to start the creation of an element. The use case is I have a custom toolbar that I want to show the Delete button for whenever a marker is selected. However for text elements, you can be in a state where the text is rendered, and the user is typing, but neither `markercreate` nor `markerselect` has been triggered

<img width="321" alt="image" src="https://user-images.githubusercontent.com/1629901/186752986-6549ae34-b9b1-4300-853a-ec59f167ae27.png">

basically i'd like to show a delete button to abort the text element during creation

**Comment by ailon** [comment]
I could definitely add an event that is fired on the first "mouse down", however enabling that delete button sounds like the only feasible use case for it. And not a very major at that :)

**Comment by tarngerine** [comment]
Yea that's fair, it's not a huge deal, just curious! Can close.
